openpower/configs/hostboot/vesnin.config

```diff
@@ -0,0 +1,64 @@
+# The Serial Flash Controller is the AST2400 BMC.
+set   SFC_IS_AST2400
+set   BMC_DOES_SFC_INIT
+unset SFC_IS_IBM_DPSS
+set   ALLOW_MICRON_PNOR
+set   ALLOW_MACRONIX_PNOR
+
+# VPD options.
+set MVPD_READ_FROM_HW
+set MVPD_WRITE_TO_HW
+set MVPD_READ_FROM_PNOR
+set MVPD_WRITE_TO_PNOR
+set DJVPD_READ_FROM_HW
+set DJVPD_WRITE_TO_HW
```

Comment:
FYI on the DJVPD_WRITE_TO_HW flag.  We have heard that some of our partners don't like this set.  Basically this allows us to store lane repair information for bad memory bits.  If we write into the DIMM then the repair will follow the DIMM around.  We like that for our systems but some other folks don't, especially in early bringup of the system where you might hit false errors due to board issues.

Reply:
Thanks, we will keep it in mind.